Pocket/pocket-ios
# Search Saves via Siri or App Shortcuts

## Goal
* This PR adds an app Intent for Searching articles in your Saves list

> [!NOTE]
> As of now, the search is intentionally limited to `Saves` (not `Archive` nor `All`)

## Test Steps
* Build/run this branch
* Add the "Search Pocket" app shortcut to your shortcut, then run it or
* Tell Siri "Search Pocket" or "Show me some cool stuff in Pocket"
* Input the search criteria (via text or voice) and make sure the search triggers in Pocket and displays results accordingly

## Issue(s)
POCKET-10508

## Video
<div align="center">
  <video src="https://github.com/user-attachments/assets/8c6d0ea0-984f-4d61-b12d-03c408f63863"/>
</div>
